core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.component.ts

```diff
@@ -129,6 +135,57 @@ export class TopicsListComponent {
     }, () => {});
   }
 
+  /**
+   * @returns {boolean} Checks whether the SerialChapterLaunch feature
+   * flag is enabled.
+   */
+  isSerialChapterLaunchFeatureEnabled(): boolean {
+    return this.platformFeatureService.status.
+      SerialChapterLaunchCurriculumAdminView.isEnabled;
+  }
+
+  ngOnInit(): void {
+    if (this.isSerialChapterLaunchFeatureEnabled()) {
+      this.TOPIC_HEADINGS = [
+        'index', 'name', 'added_stories_count', 'published_stories_count',
+        'notifications', 'subtopic_count', 'skill_count', 'topic_status'];
+    } else {
+      this.TOPIC_HEADINGS = [
+        'index', 'name', 'canonical_story_count', 'subtopic_count',
+        'skill_count', 'topic_status'
+      ];
+    }
```

Comment:
Do the headings need to be in a specific order? If not, I'd suggest separating the feature specific headings to another list. When the feature is enabled, just append that to the default list.

This will help readers understand which headers correspond to the feature.

Reply:
We will be needing the headings in the order as written there, that is why I have kept it inside the feature flag condition. I think It will be rather complex to append the fields and remove the others based on feature than to use a simple if condition here.

Reply:
Ah okay, my suggestion was only in case the heading order didn't matter in the defined list. So feel free to ignore this one.